File: Server/Data/WoahMe.Data.Models/Location.cs
namespace WoahMe.Data.Models
{
    public class Location
    {
        public int Id { get; set; }

        public int CityId { get; set; }

        public virtual City City { get; set; }

        public int GeoOrientationId { get; set; }

        public virtual GeoOrientation GeoOrientation { get; set; }
    }
}

File: Client/WoahMe/app/src/main/java/woahme/teamwork/com/woahme/Models/Location.java
package woahme.teamwork.com.woahme.Models;

public class Location {
    private String name;
    private GeoOrientation geoOrientation;

    public Location(String name, GeoOrientation orientation) {
        this.name = name;
        this.geoOrientation = orientation;
    }

    public String toString() {
        return ", " + this.name + ", " + this.geoOrientation.toString();
    }
}
